doc/install/ceph-deploy/quick-ceph-deploy.rst

```diff
@@ -205,13 +205,14 @@ Odd numbers of monitors tend to be better, although this is not required.
    it now and then push it out to each node with
    ``ceph-deploy --overwrite-conf config push {ceph-nodes}``.
 
-Add two Ceph Monitors to your cluster::
+Add two Ceph Monitors to your cluster using the command::
```

Comment:
i'd suggest remove "using the command" as it's redundant -- existing instruction is self-explained.

Reply:
Okay. Removed the redundant phrase in a new commit.